src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+	
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.BotService</PackageId>
+    <Description>Microsoft Azure Management Bot Service Library</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Is this a preview sdk? If yes, we encourage RPs to start with `0.x.x-preview` version.

Reply:
This is the first release, so it seems reasonable to release as a preview first. Will update.